assume/markets/base_market.py

```diff
@@ -245,6 +248,11 @@ def order_matches_req(o):
         )
 
     async def clear_market(self, market_products: list[MarketProduct]):
+        # Check if order is in time slots for current opening
+        # for order in self.all_orders:
```

Comment:
This should be part of `handle_orderbook` - see L153
As multiple clearings could be open in the same time, we should see if the sent orders are part of `self.open_slots = []` defined  in L52 (currently unused)